variants/heltec_esp32c3/variant.h

```diff
@@ -1,16 +1,43 @@
-#define BUTTON_PIN 9
-
-// LED pin on HT-DEV-ESP_V2 and HT-DEV-ESP_V3
-// https://resource.heltec.cn/download/HT-CT62/HT-CT62_Reference_Design.pdf
-// https://resource.heltec.cn/download/HT-DEV-ESP/HT-DEV-ESP_V3_Sch.pdf
-#define LED_PIN 2 // LED
-#define LED_INVERTED 0
+/*
+ * Heltec HT-CT62, Heltec HT-DEV-ESP V2 and Heltec HT-DEV-ESP V3
+ * Heltec HT-CT62 Schematic: https://resource.heltec.cn/download/HT-CT62/HT-CT62_Reference_Design.pdf
+ * Heltec HT-DEV-ESP V3 Schematic: https://resource.heltec.cn/download/HT-DEV-ESP/HT-DEV-ESP_V2_Sch.pdf
+ * Heltec HT-DEV-ESP V3 Schematic: https://resource.heltec.cn/download/HT-DEV-ESP/HT-DEV-ESP_V3_Sch.pdf
+ */
 
 #define HAS_SCREEN 0
+
```

Comment:
If you define I2C pins, why not activate the oled screen code?

Reply:
Not sure that anyone using this particular board would want to hook up a oled, way easier to spend the extra couple of dollars and get the Heltec V3 with the screen included.

Reply:
> If you define I2C pins, why not activate the oled screen code?

Good point, will enable it then. I'm assuming the BLE pairing PIN generation checks to see if the OLED was actually initialized then.